Gemfile

```diff
@@ -28,9 +28,13 @@ gem 'jbuilder', '~> 2.7'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.4', require: false
 
+gem 'devise'
+
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'factory_bot_rails'
+  gem 'faker'
```

Comment:
Please specify the gems versions (Check the Gemfile.lock), .i.e.

```ruby

gem 'devise', '4.8.0'`

....
```

Reply:
Done c26ad2f822a81bf5e46f17b5d59eea99a8843fd2